src/site/markdown/server/manage-guice-distributed-james.md

```diff
@@ -0,0 +1,141 @@
+details categorizing the slowness are appreciated as well (details of the slow requests for instance).
+
+### WebAdmin HealthChecks
+
+James webadmin API allows to run healthChecks for a quick health overview.
+
+Here is related [webadmin documentation](manage-webadmin.html#HealthCheck)
+
+Here are the available checks alongside the insight they offer:
+
+ - **Cassandra backend**: Cassandra storage. Ensure queries can be executed on the connection James uses.
+ - **ElasticSearch Backend**: ElasticSearch storage. Triggers an ElasticSearch health request on indices James uses.
+ - **RabbitMQ backend**: RabbitMQ messaging. Verifies an open connection and an open channel are well available.
+ - **Guice application lifecycle**: Ensures James Guice successfully started, and is up. Logs should contain explanations if
+ James did not start well.
+ - **MessageFastViewProjection**: Follows MessageFastViewProjection cache miss rates and warns if it is below 10%. If this 
+ projection is missing, this results in performance issues for JMAP GetMessages list requests. WebAdmin offers a
+ [global](manage-webadmin.html#recomputing-global-jmap-fast-message-view-projection) and 
+ [per user](manage-webadmin.html#recomputing-user-jmap-fast-message-view-projection) projection re-computation.
+
+## Mail Processing
+
+Mail processing allows to take asynchronously business decisions on received emails.
+
+Here are it's components:
+
+ - The `spooler` takes mail out of the mailQueue and executes mail processing within the `mailet container`.
+ - The `mailet container` synchronously execute the user defined logic. This 'logic' is written though the use of `mailet`,
+ `matcher` and `processor`.
+ - A `mailet` represent an action: mail modification, envelop modification, a side effect, or stop processing.
+ - A `matcher` represent a condition to execute a mailet.
+ - A `processor` is a flow of pair of `matcher` and `mailet` executed sequentially. The `ToProcessor` mailet is a `goto` 
+ instruction to start executing another `processor`
+ - A `mail repository` allow storage of a mail as part of its processing. Standard configuration relies on the following 
+ mail repository:
+     - `cassandra://var/mail/error/` : unexpected errors that occurred during mail processing. Emails impacted by performance 
+     related exceptions, or logical bug within James code are typically stored here. These mail could be reprocessed once the 
+     cause of the error is fixed. The `Mail::error` field can help diagnose the issue. Correlation with logs can be achieved
+     via the use of the `Mail::name` field.
+     - `cassandra://var/mail/address-error/` : mail addressed to a non existing recipient of an handled local domain. These 
+     mails could be reprocessed once the user is created, for instance.
+     - `cassandra://var/mail/relay-denied/` : mail for whom relay was denied: missing authentication can for instance be a 
```

Comment:
`missing authentication can, for instance, be a cause. In addition,` 
Maybe we need some `,` I wanted particularly to make it clear